carma_wm_ctrl/src/WMBroadcasterNode.cpp

```diff
@@ -156,30 +156,34 @@ carma_ros2_utils::CallbackReturn WMBroadcasterNode::handle_on_activate(const rcl
   timer_ = create_timer(get_clock(),
                           std::chrono::milliseconds((int)(config_.traffic_control_request_period * 1000)),
                           std::bind(&WMBroadcasterNode::spin_callback, this));
-  
+
   /////////////
   //SUBSCRIBERS
   ///////////// NOTE: subscriber declaration delayed until here so that when map is received, publisher is already activated to immediately publish back
 
   // Base Map Sub
   base_map_sub_ = create_subscription<autoware_lanelet2_msgs::msg::MapBin>("base_map", 1, std::bind(&WMBroadcaster::baseMapCallback, wmb_.get(), std_ph::_1));
-  
+
   // Base Map Georeference Sub
   georef_sub_ = create_subscription<std_msgs::msg::String>("georeference", 1, std::bind(&WMBroadcaster::geoReferenceCallback, wmb_.get(), std_ph::_1));
-  
+
   // Geofence Sub
-  geofence_sub_ = create_subscription<carma_v2x_msgs::msg::TrafficControlMessage>("geofence", 200, std::bind(&WMBroadcaster::geofenceCallback, wmb_.get(), std_ph::_1));
-  
+  rclcpp::SubscriptionOptions geofence_sub_options;
+  geofence_sub_options.use_intra_process_comm = rclcpp::IntraProcessSetting::Disable;
+  auto sub_qos_transient_local = rclcpp::QoS(rclcpp::KeepAll());
+  sub_qos_transient_local.transient_local();
+  geofence_sub_ = create_subscription<carma_v2x_msgs::msg::TrafficControlMessage>("geofence", sub_qos_transient_local, std::bind(&WMBroadcaster::geofenceCallback, wmb_.get(), std_ph::_1));
+
   // External Map Msg Sub
-  incoming_map_sub_ = create_subscription<carma_v2x_msgs::msg::MapData>("incoming_map", 20, std::bind(&WMBroadcaster::externalMapMsgCallback, wmb_.get(), std_ph::_1));
-  
+  incoming_map_sub_ = create_subscription<carma_v2x_msgs::msg::MapData>("incoming_map", 20, std::bind(&WMBroadcaster::externalMapMsgCallback, wmb_.get(), std_ph::_1), geofence_sub_options);
```

Comment:
Looks like `geofence_sub_options` was accidentally added to `incoming_map_sub_` here instead of `geofence_sub_`.

Reply:
Thanks for catching this!